Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -53,17 +46,17 @@ class BackForwardTableViewCell: UITableViewCell {
     var site: Site? {
         didSet {
             if let s = site {
-                faviconView.setFavicon(forSite: s, onCompletion: { [weak self] (color, url) in
-                    if InternalURL.isValid(url: s.tileURL) {
-                        self?.faviconView.image = UIImage(named: "faviconFox")
-                        self?.faviconView.image = self?.faviconView.image?.createScaled(CGSize(width: BackForwardViewCellUX.IconSize, height: BackForwardViewCellUX.IconSize))
-                        self?.faviconView.backgroundColor = UIColor.White
-                        return
-                    }
-
-                    self?.faviconView.image = self?.faviconView.image?.createScaled(CGSize(width: BackForwardViewCellUX.IconSize, height: BackForwardViewCellUX.IconSize))
-                    self?.faviconView.backgroundColor = color == .clear ? .white : color
-                })
+                var url = s.url
+                if InternalURL.isValid(url: s.tileURL) {
```

Comment:
Note: I think this API is misnamed. It should be called something like `isValidInternalURL` or `url(isInternal: URL) -> Bool`. Because from what is in front of me right now, it is unclear at a glance what the API does, and I have to stop and think about it.